libs/blocks/global-footer/global-footer.js

```diff
@@ -356,12 +336,17 @@ class Footer {
         </div>
       </div>`;
 
+    if (this.useFederatedContent) federatePictureSources(this.elements.footer);
```

Comment:
self-review: As the pictureSources are already being federated at the time of fetching, this should not be needed anymore.

Reply:
In which case we wouldn't even need `this.useFederatedContent`, right?

Reply:
Yes, though I'll need to check this in detail